### ISSUE TITLE
[Bug]: Newlines/linewrap does not work properly when tab is out-of-focus

### ISSUE THREAD

**Issue by YJDoc2** [wavetermdev/waveterm#952]
### Current Behavior

When running a command that produces logs, if I switch the tab (in waveterm) to some other tab, and com back to the original one newlines/linewrap is not correctly formatted in the logs produced at the time I was on some other tab. For example, if the command produces logs like
```
line1
line2
line3
line4
line5
...
```
If I switch the tab after seeing `line1` , and switch back after sometime, I am seeing logs as
```
line1
line2 line3 line4 line5 ...
...
```
Sometimes `line4` and `line5` are on a new line as expected, sometimes they are not.

### Expected Behavior

The logs of commands should get displayed like
```
line1
line2
line3
line4
line5
...
```
Irrespective if I switch tabs or not.

### Steps To Reproduce

1. Get some program which produces a lot of log lines. 
2. Start it in a cmd in a tab in a waveterm, see that logs are getting produced on individual lines as expected
3. Change the tab to some other tab, stay on that for a while
4. Switch back, now some of the log lines would be bunched in a single line instead of individual lines.

### Wave Version

0.8.8

### Architecture

None

### Questionnaire

- [ ] I'm interested in fixing this myself but don't know where to start
- [ ] I would like to fix and I have a solution
- [ ] I don't have time to fix this right now, but maybe later

**Comment by sawka** [comment]
thanks for posting this issue.  i have a mitigating fix that i've been working on.  but i have yet to figure out the real underlying cause.

**Comment by sawka** [comment]
i think this was fixed in the v0.9 builds.  let us know if you are still seeing this issue show up after updating.  at the very least, it should show up much less often.
